src/Makefile.sources

```diff
@@ -138,6 +138,7 @@ source_h = \
 	gen9_hevc_enc_utils.h \
 	gen9_hevc_encoder.h \
 	gen10_hcp_common.h \
+	i965_vdenc_defines.h     \
```

Comment:
The new structures defined in i965_vdenc_defines.h isn't used for VDEnc H264, how about to remove this file, we can add the new structures back if needed in the future.